extensions/ql-vscode/src/remote-queries/repository-selection.ts

```diff
@@ -41,13 +44,20 @@ export async function getRepositorySelection(): Promise<RepositorySelection> {
   } else if (quickpick?.repositoryList) {
     void logger.log(`Selected repository list: ${quickpick.repositoryList}`);
     return { repositoryLists: [quickpick.repositoryList] };
-  } else if (quickpick?.useCustomRepository) {
+  } else if (quickpick?.useCustomRepo) {
     const customRepo = await getCustomRepo();
     if (!customRepo || !REPO_REGEX.test(customRepo)) {
       throw new UserCancellationException('Invalid repository format. Please enter a valid repository in the format <owner>/<repo> (e.g. github/codeql)');
     }
     void logger.log(`Entered repository: ${customRepo}`);
     return { repositories: [customRepo] };
+  } else if (quickpick?.useAllReposOfOwner) {
+    const owner = await getOwner();
+    if (!owner || !OWNER_REGEX.test(owner)) {
+      throw new UserCancellationException('Invalid user or organization format. Please enter a valid user or organization (e.g. github)');
```

Comment:
I think this can be simpler:

```suggestion
      throw new Error(`Invalid user or organization: ${owner}`);
```

Also, I changed from `UserCancellationException` to `Error` since `UserCancellationException` is only for operations that a user explicitly cancels (so it won't be logged as an error).